### PR TITLE
[pt-br] Add content/pt-br/docs/reference/scheduling-policies/policies.md

### PR DESCRIPTION
**Pull request by AnaCarolinaRodriguesLeite**
Hello guys,
I'm re-correcting the https://github.com/kubernetes/website/pull/36083 with the policies name. That was under someone else's name.

### CODE REVIEW

File: content/pt-br/docs/reference/scheduling-policies/policies.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Políticas do escalonador
+content_type: concept
+sitemap:
+  priority: 0.2 # Scheduling priorities are deprecated
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/scheduling-policies/policies.md:5]: There's a missing `weight: 30` attribute at the end of this front matter content.

File: content/pt-br/docs/reference/scheduling-policies/policies.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+<!-- overview -->
+
+Nas versões anteriores à v1.23 do Kubernetes, a política do escalonador pode ser usada para especificar os processos *predicates* e *priorities*. Por exemplo, você pode definir a política de escalonador rodando os seguintes comandos:
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/scheduling-policies/policies.md:10]: A few suggestions here:
* Perhaps write "política do escalonador" on the second sentence too.
* I would reword this to include the commands inline in the sentence like the original documentation. This code block is looking very weird in the rendered version.

File: content/pt-br/docs/reference/scheduling-policies/policies.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+```
+ `kube-scheduler --policy-config-file <filename>` 
+ ou
+  `kube-scheduler --policy-configmap <ConfigMap>`.
+ ```
+
+Essa política escalonadora não é compatível desde a v1.23 do Kubernetes. Flags associadas `policy-config-file`, `policy-configmap`, `policy-configmap-namespace` e `use-legacy-policy-config` também não são compatíveis. Em vez disso, veja [configuração do escalonador](/docs/reference/scheduling/config/) para conseguir um comportamento similar.
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/scheduling-policies/policies.md:17]: Some suggestions here too:
* Use "política do escalonador" in this paragraph too.
* The original doc reads "use the scheduler configuration". I would use the verb "usar" instead of "ver" too: "use a API de configuração do escalonador...".

File: content/pt-br/docs/reference/scheduling-policies/policies.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## {{% heading "whatsnext" %}}
+
+* Descubra mais sobre o [escalonamento](/docs/concepts/scheduling-eviction/kube-scheduler/)
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/scheduling-policies/policies.md:21]: This page is translated, so this link can be updated. I would also write "escalonador" in lieu of "escalonamento".

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @AnaCarolinaRodriguesLeite! 👋 

Thanks for your contribution! 🎉 

This PR has an issue with it: this page is in the incorrect folder. It is currently residing on `content/pt-br/docs/reference/scheduling-policies`, and `scheduling-policies` is a folder that does not exist in the English version of this website. Could you please rename that folder to `scheduling`, which is the correct name?

I also left some considerations to improve readability. If you disagree, please feel free to suggest a different change. I am also available on the Kubernetes Slack should you need any help or wish to discuss the changes. 😃

I am holding this PR so it doesn't get merged with a file in the wrong folder. Feel free to unhold it once the folder issue has been addressed.

Thanks again!

/hold

File: content/pt-br/docs/reference/scheduling/policies.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+<!-- overview -->
+
+Nas versões anteriores à v1.23 do Kubernetes, a política do escalonador pode ser usada para especificar os processos *predicates* e *priorities*. Por exemplo, você pode definir a política do escalonador rodando os seguintes comandos:
+```
+ `kube-scheduler --policy-config-file <filename>` 
+ ou
+  `kube-scheduler --policy-configmap <ConfigMap>`.
+ ```
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/scheduling/policies.md:16]: This block is rendering really strangely. The fenced code block should not be here (` ``` `), as we only want the commands to be rendered as code (and not the word "ou").

File: content/pt-br/docs/reference/scheduling/policies.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+Essa política do escalonador não é compatível desde a v1.23 do Kubernetes. Flags associadas `policy-config-file`, `policy-configmap`, `policy-configmap-namespace` e `use-legacy-policy-config` também não são compatíveis. Em vez disso, use a API de [configuração do escalonador](/docs/reference/scheduling/config/) para conseguir um comportamento similar.
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/scheduling/policies.md:18]: "não é compatível desde a v1.23 do Kubernetes" is sounding a little strangely to me. Perhaps "não é compatível desde o Kubernetes v1.23" would sound better?

File: content/pt-br/docs/reference/scheduling/policies.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+## {{% heading "whatsnext" %}}
+
+* Descubra mais sobre o [escalonador](content/pt-br/docs/concepts/scheduling-eviction/kube-scheduler/)
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/scheduling/policies.md:22]: This link is currently broken. It should not be prefixed by `content`. Instead, it should start with `/pt-br/`. Example: https://github.com/kubernetes/website/blob/c44bdb2217adab9ce05f7b478f5cbd143b5640f2/content/pt-br/docs/concepts/configuration/secret.md?plain=1#L110

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @AnaCarolinaRodriguesLeite! 👋 

Thank you very much for your contribution! 🎉 

Found some minor bits to be addressed. If you have any questions, please don't hesitate to reach out.

File: content/pt-br/docs/reference/scheduling/policies.md (hunk 6 above)
Comment (devlware) [reply content/pt-br/docs/reference/scheduling/policies.md:18]: Or: "Desde a versão 1.23 do Kubernetes esta política não é mais compatível"

**Comment by devlware** [comment]
@AnaCarolinaRodriguesLeite thank you for your contribution, after fixing the points that Mauren suggested this is lgtm. :)

**Comment by stormqueen1990** [comment]
Hello there, @AnaCarolinaRodriguesLeite! 👋

Thank you very much for your work on this PR. However, it has been open and reviewed for a while with no action taken. For this reason, I will be closing this PR. If you wish to continue your work on this, please feel free to reopen it.

Should you have any questions, please don't hesitate to reach out to the `#kubernetes-docs-pt` channel in Kubernetes Slack.

---

Olá, @AnaCarolinaRodriguesLeite! 👋

Obrigada pelo teu trabalho neste PR. Devido à inatividade após a revisão, irei fechá-lo. No entanto, caso desejes continuar teu trabalho, podes reabrir o PR a qualquer momento.

Se tiveres dúvidas, não hesita em contatar-nos através do canal `#kubernetes-docs-pt` no Slack do Kubernetes.

/close
